content/en/docs/contribute/intermediate.md

```diff
@@ -544,13 +608,13 @@ serve it locally.
 
 Alternatively, you can install and use the `hugo` command on your development machine:
 
-1.  [Install Hugo](https://gohugo.io/getting-started/installing/) version {{< hugoVersion >}} or later.
+1.  [Install Hugo](https://gohugo.io/getting-started/installing/) version {{< hugoVersion >}}.
 
 2.  In a terminal, go to the root directory of your clone of the Kubernetes
     docs, and enter this command:
 
       ```bash
-      hugo server
+      hugo serve
```

Comment:
small nit, @aimeeu . Yes, `hugo serve` I believe does work, but the Hugo pages refer to:
https://gohugo.io/commands/hugo_server/#readout
Also, the k8s/website/Makefile has a server target that calls `hugo server` as well as the make targets to run `docker-serve`. The k8s/website/README.md has more details.

Reply:
ooops. I've been using `hugo serve` since before I started here, learned from an online Hugo class. Thanks for pointing out the correct usage.